InteractiveHtmlBom/Run.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+echo -------------------------------------------------------------------------------------------------------------------
+echo -------------------------------------------------------------------------------------------------------------------
+echo                                                                                                                                                                                    -
+echo                                  Thankyou For Using Generate InteractiveBom
+echo                                              Powered By Qu1ck
+echo                                Bat Version : Powered By Scarrrr0725
+echo                                                                                                                                                                                    -
+echo --------------------------------------------------------------------------------------------------------------------
+echo --------------------------------------------------------------------------------------------------------------------
+
+::delete --show-dialog after frist sart up and setting
+set option=--show-dialog
+
+set FilePath=%~dp0
+set pyFilePath=%FilePath%generate_interactive_bom.py
+
+:convert
+set /p pathofEDASourceFile=Please Drag the EasyEDA PCB source file here :
+
+echo  Converting. . . . . . . . . .
+python %pyFilePath% %pathofEDASourceFile% %option%
+echo %pyFilePath% %pathofEDASourceFile%%option%
```

Comment:
There should be a space here before %option% same as on the line above.

Reply:
oh........I fogot to delete this line，it used to test my bat file